cloud/docker/_docker.py

```diff
@@ -1853,7 +1855,7 @@ def main():
             volumes_from    = dict(default=None, type='list'),
             links           = dict(default=None, type='list'),
             devices         = dict(default=None, type='list'),
-            memory_limit    = dict(default=0, type='int'),
+            memory_limit    = dict(default=0),
```

Comment:
this should be explicit type either `string` or `raw`, but I'll update after merge